src/main/scala/wdl4s/wdl/WdlCall.scala

```diff
@@ -129,7 +163,7 @@ sealed abstract class WdlCall(val alias: Option[String],
       val throwables = errors.toList map { _._2.failed.get }
       Failure(ValidationException(s"Input evaluation for Call $fullyQualifiedName failed.", throwables))
     } else {
-      Success(success map { case (d, v) => d -> v.get })
+      Success(success mapValues { _.get })
```

Comment:
`mapValues` gives you a view disguised as a copy. You can do `mapValues(...).view.force` to get a copy.